bors.toml

```diff
@@ -4,8 +4,8 @@
 # Gate PRs on the `full` CircleCI workflow
 status = ["full"]
 
-# Wait until the `commit` CircleCI workflow passes before queueing a PR
-pr_status = ["commit"]
+# Wait until the `commit` GHA workflow passes before queueing a PR
+pr_status = ["Commit / %"]
```

Comment:
I'm not sure if this is what we want:

* If we have only one job, the wildcard is not needed.
* If we have more than one job, there could be cases where *at a point in time* all jobs are green just because other ones have not started yet (with job dependencies).

I'd either hardcode `Commit / Checks`, or create a "build finished" job like we do elsewhere using bors.